salt/elife-bot/config/mockserver-elife-bot.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+echo "elife-bot expectations"
+
+curl -v -X PUT "http://app:1080/mockserver/expectation" -d '{
```

Comment:
So, the machinery works like this:
- a `app` image runs `mockserver` and expose 1080 to other containers
- a second `configure` container runs at startup to initialize it
- the `configure` container uses a PUT-based API to place expectations on `app:1080`
- the port is exposed to the host on `2080` because `1080` is common and was already allocated to `mailcatcher`

I agree we should add a constraint that requires the bot's request to be a `POST`, right now this will accept any GET/POST/PUT/DELETE/etc.